firmware/Core/Inc/eps/eps_config.h

```diff
@@ -0,0 +1,11 @@
+#ifndef __EPS_CONFIG_H
+#define __EPS_CONFIG_H
+
+#include <stdint.h>
+#include "../config/configuration.h"
+
+extern const uint32_t test_integer_config_eps = 1; // const at compile time
+
+extern char test_string_config_eps[10] = "test\0";
```

Comment:
Please remove `\0` from end of string. A null terminator is placed at the end of string literals implicitly.